[BUILDING] education contribution game (#870)
zkSNACKs/WasabiDoc: docs/building-wasabi/ContributionGame.md

The bounty of `1000 US Dollars` worth of bitcoin is paid out every month on the 25th.
The depth, quality and continuous maintenance of this documentation is a proof of the success of this contribution game!

> The documentation contribution game got expanded to the education contribution game in June 2020.

Reviewer: Maybe we should say `got replaced by` instead of `got expanded to` to be clear?
```suggestion
The documentation contribution game got replaced by the education contribution game in June 2020.
```
